subsys/shell/Kconfig

```diff
@@ -25,6 +25,13 @@ config SHELL_MINIMAL
 	  defaults which favor reduced flash or memory requirements over extra
 	  features.
 
+config SHELL_THREAD_PRIORITY
+	int "Shell thread priority"
+	default 99
+	range 0 99
```

Comment:
What is the rationale behind this?

The lowest priority possible is `MAX(CONFIG_NUM_PREEMPT_PRIORITIES) - 1`, which is `127`.

https://github.com/zephyrproject-rtos/zephyr/blob/a5fce2022aa00f145ef08e9f876c4f9ee416e4ac/include/kernel.h#L53

https://github.com/zephyrproject-rtos/zephyr/blob/a5fce2022aa00f145ef08e9f876c4f9ee416e4ac/include/kernel.h#L50

https://github.com/zephyrproject-rtos/zephyr/blob/4084242a71d0a93d97a83776b995785bd28ec883/kernel/Kconfig#L55-L59

Reply:
one of our google projects needs to be able to run the shell thread at a higher priority.

Reply:
what are the next steps? how to merge this CL?